addons/website/static/src/scss/website.scss

```diff
@@ -1160,3 +1160,9 @@ table.table_desc tr td {
 .modal-footer > .float-left {
     margin-right: auto;
 }
+
+.o_older_container {
```

Comment:
Why "older" container? I don't understand the name, could you explain?

Reply:
Indeed, bad naming, should be `o_mega_menu_container`